components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketStory.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket
+
+/**
+ * A Pocket story downloaded from the Internet and intended to be displayed in the application.
+ */
+sealed class PocketStory {
+    /**
+     * Title of the story.
+     */
+    abstract val title: String
+
+    /**
+     * Url where the story can be full read.
+     */
+    abstract val url: String
+
+    /**
+     * A Pocket recommended story.
+     *
+     * @property title the title of the story.
```

Comment:
```suggestion
     * @property title The title of the story.
```
Let's capitalize since they're a sentence.

Reply:
Thank you. Will do that for all properties here.